static/js/components/DiscussionCard.js

```diff
@@ -0,0 +1,25 @@
+// @flow
+import React from 'react';
+import { Card } from 'react-mdl/lib/Card';
+import Icon from 'react-mdl/lib/Icon';
+
+import type { Program } from '../flow/programTypes';
+
+type DiscussionCardProps = {
+  program: Program
+};
+
+const DiscussionCard = ({program}: DiscussionCardProps) =>
+  <Card className="discussion-card" shadow={0}>
```

Comment:
Could you gate this with a feature flag so we can have it hidden until OD goes live?

Reply:
yeah I think we should do that too, will do